perun-core/src/main/java/cz/metacentrum/perun/core/impl/VosManagerImpl.java

```diff
@@ -358,5 +384,124 @@ public int getVosCount(PerunSession sess) {
 		}
 	}
 
+	@Override
+	public BanOnVo setBan(PerunSession sess, BanOnVo banOnVo) {
+		Utils.notNull(banOnVo.getValidityTo(), "banOnVo.getValidityTo");
+		try {
+			int newId = Utils.getNewId(jdbc, "vos_bans_id_seq");
+
+			jdbc.update("insert into vos_bans(" +
+							"id, " +
+							"description, " +
+							"banned_to, " +
+							"member_id, " +
+							"vo_id, " +
+							"modified_by, " +
+							"modified_at, " +
```

Comment:
Shouldn't you also set "created_by" and "created_at"?

Reply:
Yes, I should add the `created_by`, but the `created_at` and `modified_at` are not needed because they will have a default value set in DB. I have added the `created_by` and removed `modified_at`.